src/CharterManager.sol

```diff
@@ -229,6 +242,28 @@ contract CharterManagerImp {
         VatLike(vat).move(urp, dst, rad);
     }
 
+    function roll(bytes32 srcIlk, bytes32 dstIlk, address src, address dst, int256 srcDart) external allowed(src) allowed(dst) {
```

Comment:
Gonza mentioned changing `srcDart` to a `uint256` in chat, and I think I agree with him. `int256` is a little more generalized, but it's practically a confusing parameter. Additionally even if we allow positive `srcDart` for a reverse transfer (I'm assuming this is the intent?) we probably want to have explicit permission set for rebalancing in either direction. If we allow positive `srcDart` then `rollable` permissions only need to be set in one direction and you get both.

Reply:
Ok, changed `srcDart` to uint256.
The original intention was too keep it similar to `frob`, as now we have `dart`s with different types in different functions.
But it's probably ok.